raiden/transfer/state_change.py

```diff
@@ -831,3 +1152,14 @@ def __eq__(self, other):
 
     def __ne__(self, other):
         return not self.__eq__(other)
+
+    def to_dict(self):
```

Comment:
why implement this as you don't implement it for the class above and it is computed automatically anyway?